packages/legacy-scripting-runner/index.js

```diff
@@ -1071,6 +1072,7 @@ const legacyScriptingRunner = (Zap, zcli, input) => {
       params.client_id = clientId;
       params.client_secret = clientSecret;
       params.refresh_token = bundle.authData.refresh_token;
+      body.redirect_uri = bundle.inputData.redirect_uri;
```

Comment:
Should we use `params` here?

```suggestion
      params.redirect_uri = bundle.inputData.redirect_uri;
```

Reply:
ah yeah, probably! good call, i'll fix.